tools/pythonpkg/tests/fast/test_pytorch.py

```diff
@@ -34,7 +34,8 @@ def test_pytorch():
         torch.equal(duck_torch['a'], torch.tensor(duck_numpy['a']))
         torch.equal(duck_torch['b'], torch.tensor(duck_numpy['b']))
 
-    with pytest.raises(TypeError, match="can't convert"):
-        con = duckdb.connect()
-        con.execute(f"create table t( a UINTEGER)")
-        duck_torch = con.sql("select * from t").torch()
+    # Comment out test that might fail or not depending on pytorch versions
+    # with pytest.raises(TypeError, match="can't convert"):
```

Comment:
We can add a `skipif` instead, that will test the version of pytorch, similar to this:
```py
    @pytest.mark.skipif(
        Version(pd.__version__) < Version('2.0.2'), reason="pandas < 2.0.2 does not properly convert timezones"
    )
```
(source: `tools/pythonpkg/tests/fast/pandas/test_datetime_timestamp.py`)

Reply:
Hmm I agree that commenting this out makes sense here, because yes this is brittle and is now testing an old error that PyTorch used to throw
But signals like this are still valuable, these limitations stopped us from considering pytorch from being a viable output format, with this test failing that tells us that those limitation(s) are now gone